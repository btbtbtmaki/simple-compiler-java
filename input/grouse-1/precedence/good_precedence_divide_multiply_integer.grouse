main {

  print 87 / 33 * 28 nl;
  print 74 / 16 * 46 nl;
  print 89 / 59 * 47 nl;
  print 8 / 69 * 15 nl;
  print 45 / 40 * 5 nl;
  print 52 / 89 * 32 nl;
  print 75 / 87 * 2 nl;
  print 79 / 6 * 86 nl;
  print 52 / 21 * 83 nl;
  print 93 / 21 * 44 nl;
  print 83 / 26 * 40 nl;
  print 62 / 2 * 4 nl;

  print 49 * 62 / 58 nl;
  print 18 * 69 / 92 nl;
  print 51 * 31 / 67 nl;
  print 51 * 88 / 19 nl;
  print 42 * 1 / 94 nl;
  print 75 * 41 / 54 nl;
  print 64 * 7 / 37 nl;
  print 88 * 69 / 19 nl;
  print 83 * 47 / 73 nl;
  print 63 * 19 / 24 nl;
  print 49 * 1 / 96 nl;
  print 32 * 3 / 38 nl;

  print 53 / 21 * 69 / 94 nl;
  print 59 / 4 * 93 / 63 nl;
  print 84 / 25 * 96 / 89 nl;
  print 3 / 99 * 82 / 32 nl;
  print 20 / 87 * 88 / 71 nl;
  print 25 / 28 * 21 / 36 nl;
  print 52 / 44 * 47 / 85 nl;
  print 43 / 46 * 43 / 1 nl;
  print 16 / 78 * 27 / 5 nl;
  print 24 / 88 * 96 / 43 nl;
  print 32 / 61 * 28 / 56 nl;
  print 53 / 97 * 98 / 55 nl;

  print 35 * 15 / 2 * 16 nl;
  print 41 * 15 / 64 * 37 nl;
  print 46 * 31 / 50 * 33 nl;
  print 70 * 89 / 47 * 94 nl;
  print 4 * 19 / 51 * 1 nl;
  print 73 * 86 / 29 * 55 nl;
  print 9 * 13 / 13 * 3 nl;
  print 44 * 62 / 33 * 55 nl;
  print 86 * 60 / 68 * 52 nl;
  print 56 * 11 / 66 * 7 nl;
  print 27 * 42 / 72 * 42 nl;
  print 19 * 33 / 12 * 51 nl;

}

