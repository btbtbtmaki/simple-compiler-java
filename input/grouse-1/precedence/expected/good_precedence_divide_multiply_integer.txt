56
184
47
0
5
0
0
1118
166
176
120
124
52
13
23
236
0
56
12
319
53
49
0
2
1
20
3
0
0
0
0
0
0
0
0
0
4192
333
924
12408
1
11880
27
4510
3900
63
630
2652
